pkg/administration/events/hints.go

```diff
@@ -1,25 +1,127 @@
+environment variable, and increase the default timeout of 2 seconds.
+`,
+				codes.AWSSHCacheExhausted: `The cache of alerts to-be-uploaded to AWS Security Hub is increasing.
+This will lead to an increasing delay in alerts being uploaded to AWS Security Hub.
+
+You may adjust the upload interval for uploading alerts to AWS Security Hub by adjusting the "ROX_AWSSH_UPLOAD_INTERVAL"
+environment variable, and lower the default interval of 15 seconds.`,
+				codes.EmailGeneric: `An issue occurred when using the Email notifier.
+Please ensure that:
+- Configuration is valid, specifically the auth information and TLS configuration.
+- Central can access the SMTP endpoint.
+
+For additional information, consult the event message for detailed information.`,
+				codes.JIRAGeneric: `An issue occurred when creating an issue via the JIRA notifier.
+Please ensure that:
+- Configuration is valid, specifically the auth information.
+- Central can access the JIRA endpoint.
+
+For additional information, consult the event message for detailed information.`,
+				codes.PagerDutyGeneric: `An issue occurred when using the PagerDuty notifier.
+Please ensure that:
+- Configuration is valid, specifically the auth information.
+- Central can access the PagerDuty endpoint.
+
+For additional information, consult the event message for detailed information.`,
+				codes.SlackGeneric: `An issue occurred when using the Slack notifier.
+Please ensure that:
+- Configuration is valid, specifically the auth information.
+- Central can access the Slack endpoint.
+
+For additional information, consult the event message for detailed information.`,
+				codes.SyslogGeneric: `An issue occurred when using the Syslog notifier.
+Please ensure that:
+- The message format is valid.
+- Central can access the Syslog endpoint.
+
+In case a timeout error occurred, you may adjust the timeout for sending alerts to Syslog by adjusting the "ROX_SYSLOG_TIMEOUT"
+environment variable, and increase the default timeout of 5 seconds.`,
+				codes.SplunkGeneric: `An issue occurred when using the Splunk notifier.
+Please ensure that:
+- Configuration is valid, specifically the Splunk HEC's vadility.
+- Central can access the Splunk endpoint.
+
+For additional information, consult the event message for detailed information.`,
+				codes.SumoLogicGeneric: `An issue occurred when using the Sumo Logic notifier.
+Please ensure that:
+- Configuration is valid, specifically the TLS configuration.
+- Central can access the Sumo logic endpoint.
+
+For additional information, consult the event message for detailed information.`,
+				codes.TeamsGeneric: `An issue occurred when using the Teams notifier.
+Please ensure that:
+- Configuration is valid.
+- Central can access the Teams endpoint.
+
+In case a timeout error occurred, you may adjust the timeout for sending alerts to teams by adjusting the "ROX_TEAMS_TIMEOUT"
+environment variable, and increase the default timeout of 10 seconds.`,
```

Comment:
Is there a reason for the different timeouts? In general I'd say 5 seconds is on the lower side, I'd just increase the default to something like 30 seconds everywhere.

Reply:
I don't really have a good answer, those have been the default timeouts since ages.

At the very least now those are configurable, as opposed to before where they weren't.

Since no-one has complained thus far, I wouldn't change them now. Either no-one uses those integrations, or the current timeout fits. If we have evidence that this isn't the case, we can always change them.

Reply:
Yeah didn't mean that we should change them in this PR. But in my experience there is no way Slack and Jira respond within 5 seconds all the time...